<launch>

  <node name="recognizer" pkg="pocketsphinx" type="recognizer.py">
    <param name="lm" value="$(find pi_speech)/params/nav_commands.lm"/>
    <param name="dict" value="$(find pi_speech)/params/nav_commands.dic"/>
  </node>
  
  <node name="voice_nav" pkg="pi_speech" type="voice_nav.py" output="screen">
  	<param name="max_speed" value="0.4"/>
  	<param name="start_speed" value="0.1"/>
  	<param name="linear_increment" value="0.05"/>
  	<param name="angular_increment" value="0.1"/>  	
  </node>

</launch>
